redaxo/src/addons/backup/lib/backup.php

```diff
@@ -132,17 +131,19 @@ public static function importDb($filename)
         $lines = [];
         rex_sql_util::splitSqlFile($lines, $conts, 0);
 
+        $error = [];
+
         $sql = rex_sql::factory();
         foreach ($lines as $line) {
             try {
                 $sql->setQuery($line['query']);
             } catch (rex_sql_exception $e) {
-                $error .= "\n" . nl2br(rex_escape($e->getMessage()));
+                $error[] = nl2br(trim(rex_escape($e->getMessage())));
```

Comment:
Ob das nl2br taugt müsste man testen. Ansonsten lgtm

Reply:
Ja, ist für mich auch nicht 100% klar, wie es besser ist. Aber so werden eben Zeilenumbrüche in der Orignalquery beibehalten in der Errormessage.